plugins/rbac/src/components/ConditionalAccess/const.ts

```diff
@@ -18,3 +20,6 @@ export const conditionButtons = [
   { val: criterias.anyOf, label: criteriasLabels[criterias.anyOf] },
   { val: criterias.not, label: criteriasLabels[criterias.not] },
 ];
+
+export const rbacAPIDocLink =
+  'https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html-single/administration_guide_for_red_hat_developer_hub/index#ref-rbac-rest-api-endpoints_assembly-rhdh-integration-aks';
```

Comment:
this link has a version hardcoded in it so with each release this would need an update and it doesn't take me to RBAC doc as well , if we just need to point to product doc then can use https://docs.redhat.com/en/documentation/red_hat_developer_hub

Reply:
@ShiranHi  @invincibleJai , this link is for the CLI keyword in the message "Multiple levels of nested conditions are not supported. Only one level is displayed. Please use the [CLI](url) to view all nested conditions". If it's just a generic link to the product doc, do we still need to put a link? If we want to point to a more specific CLI related page, it will have version in the URL which is hardcoded.

Reply:
@ciiay do we cover this in the documentation? I couldn't find it. If we do, we can use the link Jai mentioned. If not, I suppose we can just remove it since users initially configured the nested condition using the CLI, so they should know where to go back and check. Is that correct?

Reply:
@ShiranHi Agreed. In the documentation, it mentions "nested condition" in an example, but nowhere mentions the "multiple levels of nested condition are not supported" limitation. So we can just simply remove it as you suggested.